Find a functionally equivalent code:android.view.ViewGroup.dispatchDetachedFromWindow:COMMENT
Method Modifier: 
Comment:/**
 * {@inheritDoc}
 */

@@ -20,24 +20,25 @@
     for (int i = 0; i < count; i++) {
         children[i].dispatchDetachedFromWindow();
     }
+    clearDisappearingChildren();
     super.dispatchDetachedFromWindow();
 }

